src/pandroid/app/src/main/java/com/panda3ds/pandroid/C.java

```diff
@@ -0,0 +1,6 @@
+package com.panda3ds.pandroid;
+
+public class C {
```

Comment:
What is this class for? Is it just for string literals? Please give a more descriptive name

Reply:
to save global constant values